src/renderer/components/sidebar/SidebarComponent.tsx

```diff
@@ -104,6 +110,17 @@ export const SidebarComponent: React.FC<Props> = (props): JSX.Element => {
         path: walletRoutes.base.path(),
         icon: WalletIcon
       },
+      {
+        key: TabKey.SWAP,
+        label: intl.formatMessage({ id: 'common.swap' }),
+        path: poolsRoutes.swap.path({
+          source: 'BTC.BTC',
```

Comment:
can use assetToString helper here like elsewhere in the app.
assetToString(AssetBTC)